### ISSUE TITLE
Feature Request: Allow to set shell thickness

### ISSUE THREAD

**Issue by zdar** [slic3r/Slic3r#1405]
Hi,
I'd would be nice to be able to set arbitrary shell thicknes instead of setting number of layers/perimentrs.

For example to set horizontal shells to 1mm and vertical shells to 1.5mm --> Number of layers/perimetrs would be calculated by slic3r automatically depending on layer and extruder properties.

**Comment by lordofhyphens** [comment]
What does this solve other than porting a issue Cura has with having to warn you when it's not cleanly divisible over to Slic3r? Also, unless you fix the perimeter width there's a ton of possible combinations and no good heuristic that I can think of offhand to select the correct one.

**Comment by alexrj** [comment]
I tend to agree with considerations by @lordofhyphens, but I've seen that it might be useful to have this in addition to perimeters/top/bottom, defined as "Minimum Shell Thickness". Number of perimeters would be increased until that thickness is achieved, regardless of their divisibility. This option would be useful to display in the plater overrides/shortcuts.

**Comment by curieos** [comment]
I'm looking into this. I've added the option in the UI and I'm working on some checks for spiral vase.

**Comment by curieos** [comment]
I created a pull request for this feature

**Comment by lordofhyphens** [comment]
Merged PR is incomplete; sets minimum shell only.
